cinema/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=100)
+    description = models.TextField()
+    duration = models.IntegerField()
```

Comment:
Consider using `PositiveIntegerField` for the `duration` field to ensure that only positive values are accepted, as a movie duration cannot be negative. This aligns with the migration file, which uses `PositiveIntegerField`.